lib/score.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'scorecard'
+require_relative 'round'
+
+class Score 
+
+    attr_reader :first_bowl, :second_bowl, :total_score, :score_card
+
+    def initialize
+        @score_card = []
+        @total_score = 0
+    end
+
+    def total(x, y)
+    @score_card << [x, y]
+    if @score_card.length > 1
+        if @score_card[-2][0] == 10
+            strike(x,y)
+        elsif @score_card[-2].sum == 10
+            @total_score += x
+            @total_score += x + y
+        else
+            p @score_card[-2]
+            @total_score += (x + y)
+        end
+    else 
+        @total_score += (x + y)
+    end
+    p "Your current total is #{@total_score}"
+    end
+
+    def strike(x, y)
+        #IF THREE IN A ROW, ADD THIRTY
+        if @score_card.length >= 3 && @score_card[-1][0] == 10 && @score_card[-2][0] == 10 && @score_card[-3][0] == 10
```

Comment:
Perhaps drawing out a frame class could help with simplifying this logic a bit. 

It's hard to read this logic, because something like @score_card[-1][0] is difficult to understand if you don't have context from writing it - it's likely also difficult for the developer who wrote it to keep tabs on. 

If you instead used a Frame class to track the value of each roll, and then manipulated an array of Frames instead in order to tally up the score, it might make this slightly more readable, thus easier to refactor.